unyt/_physical_ratios.py

```diff
@@ -122,6 +122,11 @@
 kelvin_per_rankine = 5.0 / 9.0
 watt_per_horsepower = 745.69987158227022
 erg_per_s_per_watt = 1e7
+J_per_BTU = 1055.0559
+watt_per_hour = 3600
```

Comment:
this should be called joules_per_watt_hour I think? W/hr isn't dimensionless so calling it that doesn't make sense in this file, which only contains dimensionless ratios.

Reply:
Yep, that makes a lot more sense!